quick-start.md

```diff
@@ -13,15 +13,61 @@ has_toc: false
 
 ### Overview
 
-Open Horizon is a middleware solution designed to manage the deployment and service software lifecycle of containerized applications. It also supports an optional separation of concerns between a service and related machine learning assets by providing a separate model sync service.  These solutions enable autonomous management of large fleets of edge compute nodes in various connected states.
+Open Horizon is a middleware solution designed to manage the deployment and service software lifecycle of containerized applications. It also supports an optional separation of concerns between a service and related machine learning assets by providing a separate [model sync service](https://open-horizon.github.io/docs/developing/model_management_details/#model_management_details).  These solutions enable autonomous management of large fleets of edge compute nodes in various connected states.
```

Comment:
The URL should be relative `../developing/model_management_details/#model_management_details` so that the page loads in the OH Docs website , not take the reader to GitHub.

Reply:
fixed